apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -39,8 +39,14 @@ const TeacherHomepage = React.createClass({
         <ProtectedStatefulDiv
           ref="termsReminder"
         />
-        <Announcements
-          announcements={announcements}
+        <Notification
+          type="bullhorn"
+          notice={announcements[0].heading}
```

Comment:
Good question.  For now I left the array of announcements intact, in case we decide we want to show more than one, and I think it might be valid to say that we're displaying a single announcement as a Notification.